src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -531,36 +532,8 @@ public static PublishPackageViewModel FromLocalPackage(DynamoViewModel dynamoVie
                 License = l.License
             };
 
-            // add additional files
-            l.EnumerateAdditionalFiles();
-            foreach (var file in l.AdditionalFiles)
-            {
-                vm.AdditionalFiles.Add(file.Model.FullName);
-            }
-
             var nodeLibraryNames = l.Header.node_libraries;
```

Comment:
I'm removing this to enable package publisher the ability to remove files from their packages on subsequent versions. Right now, you only have the option to add files to a package, not remove them. By forcing you to re-add all the files, you can be more selective.

The *right* way to solve this is probably adding a "Remove File" button, though that's outside the scope of this issue.

The MeshToolkit package is live, so I no longer need this functionality, though as a package publisher, I prefer it to function with the lines removed.